readme.rst

```diff
@@ -5,13 +5,15 @@ See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-a
 
 Then go through the following sections to use this starter pack to set up your docs repository.
 
+**Note:** After setting up your repository with the starter pack, you need to track the changes to the starter pack and manually update your repository with the required files.
```

Comment:
<remove some redundancy and make it easier to read>

...you will need to track the changes made to it and manually...

Reply:
Updated (without the "will" which isn't needed imo).

Reply:
well you kinda do :smile_cat: because it's in the future (After setting up...) - not important, waiting for the CI checks...